tee-worker/ts-tests/integration-tests/substrate_ii_identity.test.ts

```diff
@@ -126,6 +127,18 @@ describe('Test Identity (direct invocation)', function () {
         */
         assert.isTrue(res.do_watch.isFalse);
         assert.isTrue(res.status.asTrustedOperationStatus[0].isInvalid);
+        assertWorkerError(
+            context,
+            requestIdentifier,
+            (v) => {
+                assert.isTrue(v.isLinkIdentityFailed, `expected LinkIdentityFailed, received ${v.type} instead`);
+                assert.isTrue(
+                    v.asLinkIdentityFailed.isUserShieldingKeyNotFound,
+                    `expected UserShieldingKeyNotFound, received ${v.asLinkIdentityFailed.type} instead`
+                );
+            },
+            res
+        );
 
         const events = await eventsPromise;
         await assertFailedEvent(context, events, 'LinkIdentityFailed', 'UserShieldingKeyNotFound');
```

Comment:
Is this check passing? If so, please disregard my previous comment about backwards compatibility, I must have misunderstood the code :)

Reply:
Tests pass 🙃